packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -1,6 +1,85 @@
 ## AWS CodeBuild Construct Library
 
-Define a project. This will also create an IAM Role and IAM Policy for CodeBuild to use.
+AWS CodeBuild is a fully managed continuous integration service that compiles
+source code, runs tests, and produces software packages that are ready to
+deploy. With CodeBuild, you don’t need to provision, manage, and scale your own
+build servers. CodeBuild scales continuously and processes multiple builds
+concurrently, so your builds are not left waiting in a queue. You can get
+started quickly by using prepackaged build environments, or you can create
+custom build environments that use your own build tools. With CodeBuild, you are
+charged by the minute for the compute resources you use.
+
+### Installation
+
+Install the module:
+
+```console
+$ npm i @aws-cdk/aws-codebuild
+```
+
+Import it into your code:
+
+```ts
+import codebuild = require('@aws-cdk/aws-codebuild');
+```
+
+The `codebuild.Project` construct represents a build project resource. See the
+reference documentation for a comprehensive list of initialization properties,
+methods and attributes.
+
+### Source
+
+Build projects are usually associated with a _source_, which is specified via
+the `source` property which accepts a class that extends the `BuildSource`
+abstract base class. The supported sources are:
+
+* `NoSource` (default, requires `buildSpec`)
+* `CodeCommitSource`
+* `S3BucketSource`
+* `CodePipelineSource`
+* `GitHubSource`
+* `GitHubEnterpriseSource`
+* `BitBucketSource`
+
+Here's an AWS CodeBuild project with no source which simply prints `Hello,
+CodeBuild!`:
+
+[Minimal Example](./test/integ.defaults.lit.ts)
```

Comment:
I would move this part of the docs to the "Using CodeBuild with other AWS services" part of the ReadMe that's below. Some reorganization probably is needed (perhaps remove the "Using CodeBuild with other AWS services" heading completely, and substitute it with this "Source" one? I'm sure you'll come up with something good 🙂).

Reply:
Okay. Reorganized a little let me know if that looks better.